src/13_file_io.py

```diff
@@ -9,10 +9,20 @@
 # Print all the contents of the file, then close the file
 
 # YOUR CODE HERE
+with open('foo.txt') as f:
+    read_data = f.read()
+    print(read_data)
+    f.closed
+
 
 # Open up a file called "bar.txt" (which doesn't exist yet) for
 # writing. Write three lines of arbitrary content to that file,
-# then close the file. Open up "bar.txt" and inspect it to make 
+# then close the file. Open up "bar.txt" and inspect it to make
 # sure that it contains what you expect it to contain
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+with open('bar.txt', 'w') as b:
+    b.write('This is a test\n')
+    b.write('More testing\n')
+    b.write('And further testing\n')
```

Comment:
You can do this all on one line. `b.write("Line one.\n Line two.\n Line three\n")`